pkg/virt-operator/resource/apply/delete_test.go

```diff
@@ -169,6 +175,77 @@ var _ = Describe("Deletion", func() {
 			Expect(needRemoved).To(BeEmpty())
 		})
 	})
+
+	Context("Node label deletion", func() {
+		var clientset *fake.Clientset
+		var kubevirtClient *kubecli.MockKubevirtClient
+		var ctrl *gomock.Controller
+
+		BeforeEach(func() {
+			ctrl = gomock.NewController(GinkgoT())
+			clientset = fake.NewSimpleClientset()
+			kubevirtClient = kubecli.NewMockKubevirtClient(ctrl)
+			kubevirtClient.EXPECT().CoreV1().Return(clientset.CoreV1()).AnyTimes()
+		})
+
+		AfterEach(func() {
+			ctrl.Finish()
+		})
+
+		It("should delete kubevirt labels from nodes", func() {
+			nodes := &k8sv1.NodeList{
+				Items: []k8sv1.Node{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "node1",
+							Labels: map[string]string{
+								"kubevirt.io/some-label": "value",
+								"other-label":            "value",
+							},
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "node2",
+							Labels: map[string]string{
+								"kubevirt.io/another-label": "value",
+							},
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:   "node3",
+							Labels: map[string]string{},
+						},
+					},
+				},
+			}
+
+			_, err1 := clientset.CoreV1().Nodes().Create(context.Background(), &nodes.Items[0], metav1.CreateOptions{})
+			Expect(err1).ToNot(HaveOccurred())
+			_, err2 := clientset.CoreV1().Nodes().Create(context.Background(), &nodes.Items[1], metav1.CreateOptions{})
+			Expect(err2).ToNot(HaveOccurred())
+			_,err3 := clientset.CoreV1().Nodes().Create(context.Background(), &nodes.Items[2], metav1.CreateOptions{})
+			Expect(err3).ToNot(HaveOccurred())
+
+
+			err := deleteKubeVirtLabelsFromNodes(kubevirtClient)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
nit, reusing variable and dropping double newline:
```suggestion
			_, err := clientset.CoreV1().Nodes().Create(context.Background(), &nodes.Items[0], metav1.CreateOptions{})
			Expect(err).ToNot(HaveOccurred())
			_, err = clientset.CoreV1().Nodes().Create(context.Background(), &nodes.Items[1], metav1.CreateOptions{})
			Expect(err2).ToNot(HaveOccurred())
			_,err = clientset.CoreV1().Nodes().Create(context.Background(), &nodes.Items[2], metav1.CreateOptions{})
			Expect(err).ToNot(HaveOccurred())

			err = deleteKubeVirtLabelsFromNodes(kubevirtClient)
			Expect(err).ToNot(HaveOccurred())
```